### PR TITLE
Postman CME Version 1.2 

### PR DESCRIPTION
**Pull request by chkp-ccurrier**
Ordered hierarchy as referenced in swaggerhub. 
Updated new fields and url to 1.2.

### CODE REVIEW

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 1]
```diff
@@ -0,0 +1,1256 @@
...
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "CME General Configuration",
+			"description" : "Operations for general CME configurations.",
+			"item": [
+					  {
+						"name": "get delayCycle",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:413]: Please change to "GET delayCycle"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 2]
```diff
@@ -0,0 +1,1256 @@
...
+						"response": []
+					  }
+					]  
+		},
+		{
+			"name" : "Management",
+			"description" : "Operations for configuring Management.",
+			"item": [
+					  {
+						"name": "GET all mgmt",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:518]: Please change to "GET management"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 3]
```diff
@@ -0,0 +1,1256 @@
...
+									"cme-api",
+									"v1",
+									"management"
+								]
+							}
+						},
+						"response": []
+					  },
+					  {
+						"name": "PUT Edit management",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:549]: Please change to "PUT set management"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 4]
```diff
@@ -0,0 +1,1256 @@
...
+						"response": []
+					  }
+					]
+		},
+		{
+			"name" : "Accounts",
+			"description" : "Operations for configuring Accounts in CME <br/>(Previously known as \"controllers\").",
+			"item": [
+					  {
+						  "name": "GET accounts",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:590]: Please change to "GET all accounts"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 5]
```diff
@@ -0,0 +1,1256 @@
...
+									  "cme-api",
+									  "v1.2",
+									  "accounts"
+								  ]
+							  }
+						  },
+						  "response": []
+					  },
+					  {
+						  "name": "GET account 'name'",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:619]: Please change to "GET account"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 6]
```diff
@@ -0,0 +1,1256 @@
...
+									  "v1.2",
+									  "accounts",
+									  "<account_name>"
+								  ]
+							  }
+						  },
+						  "response": []
+					  },
+					  {
+						  "name": "delete account",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:649]: Please change to "DELETE account"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 7]
```diff
@@ -0,0 +1,1256 @@
...
+									  "cme-api",
+									  "v1.2",
+									  "gwConfigurations"
+								  ]
+							  }
+						  },
+						  "response": []
+					  },
+					  {
+						  "name": "GET a gw configuration",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:716]: Please change to "GET gw configuration"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 8]
```diff
@@ -0,0 +1,1256 @@
...
+									  "type": "text"
+								  },
+								  {
+									  "key": "Content-Type",
+									  "value": "application/json",
+									  "type": "text"
+								  }
+							  ],
+							  "url": {
+								  "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/<gwConfigurations_name>",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:732]: Please change to ""raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/<gwConfiguration_name>","

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 9]
```diff
@@ -0,0 +1,1256 @@
...
+								  "host": [
+									  "{{server}}"
+								  ],
+								  "path": [
+									  "v1.8",
+									  "cme-api",
+									  "v1.2",
+									  "gwConfigurations",
+									  "<gwConfigurations_name>"
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:741]: Please change to "<gwConfiguration_name"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 10]
```diff
@@ -0,0 +1,1256 @@
...
+								  ]
+							  }
+						  },
+						  "response": []
+					  },
+					  {
+						  "name": "delete A gw configuration",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:748]: Please change to "DELETE gw configuration"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 11]
```diff
@@ -0,0 +1,1256 @@
...
+									  "value": "{{session}}"
+								  },
+								  {
+									  "key": "Content-Type",
+									  "type": "text",
+									  "value": "application/json"
+								  }
+							  ],
+							  "url": {
+								  "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/<gwConfigurations_name>",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:764]: Please change to "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/<gwConfiguration_name>"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 12]
```diff
@@ -0,0 +1,1256 @@
...
+								  "host": [
+									  "{{server}}"
+								  ],
+								  "path": [
+									  "v1.8",
+									  "cme-api",
+									  "v1.2",
+									  "gwConfigurations",
+									  "<gwConfigurations_name>"
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:773]: Please change to "<gwConfiguration_name"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 13]
```diff
@@ -0,0 +1,1256 @@
...
+								  },
+								  {
+									  "key": "Content-Type",
+									  "value": "application/json",
+									  "type": "text"
+								  }
+							  ],
+							  "body": {
+								  "mode": "raw",
+								  "raw": "{\"name\": \"<gwConfigurations_name>\", \"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\"}"
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:882]: Please change to "raw": "{\"name\": \"<**gwConfiguration_name**>\", \"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\"}"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 14]
```diff
@@ -0,0 +1,1256 @@
...
+									  "value": "application/json",
+									  "type": "text"
+								  }
+							  ],
+							  "body": {
+								  "mode": "raw",
+								  "raw": "{\"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\"}"
+							  },
+							  "url": {
+								  "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/azure/<gwConfigurations_name>",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:921]: Please change to "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/azure/<**gwConfiguration_name**>",

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 15]
```diff
@@ -0,0 +1,1256 @@
...
+								  "host": [
+									  "{{server}}"
+								  ],
+								  "path": [
+									  "v1.8",
+									  "cme-api",
+									  "v1.2",
+									  "gwConfigurations",
+									  "azure",
+									  "<gwConfigurations_name>"
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:931]: Please change to "<**gwConfiguration_name**>"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 16]
```diff
@@ -0,0 +1,1256 @@
...
+								"value": "application/json"
+							},
+							{
+								"key": "X-chkp-sid",
+								"value": "{{session}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"name\": \"<account_name>\", \"regions\": [\"eu-west-1\", \"us-east-2\"], \"credentials_file\": \"AWS_credentials_file_name\", \"access_key\": \"<access_key>\", \"secret_key\": \"<secret_key>\", \"sts_role\": \"<sts_role>\", \"sta_external_id\": \"<sts_external_id>\", \"scan_gateways\": \"<scan_gateways>\", \"scan_vpn\": \"<scan_vpn>\", \"scan_load_balancers\": \"<scan_load_balancers>\", \"access_subnets\": \"<access_sunbets>\", \"deletion_tolerance\": \"<deletion_tolerance>\"}",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:959]: Please change to "raw": "{\"name\": \"<account_name>\", \"regions\": [\"eu-west-1\", \"us-east-2\"], \"credentials_file\": \"AWS_credentials_file_name\", \"access_key\": \"<access_key>\", \"secret_key\": \"<secret_key>\", \"sts_role\": \"<sts_role>\", \"sts_external_id\": \"<sts_external_id>\", \"scan_gateways\": \"<scan_gateways>\", \"scan_vpn\": \"<scan_vpn>\", \"scan_load_balancers\": \"<scan_load_balancers>\", \"scan_subnets\": \"<scan_subnets>\", \"communities\": [\"<community_a>\", \"<community_b>\"], \"deletion_tolerance\": \"<deletion_tolerance>\", \"sub_accounts\": [{\"name\": \"sub_account_a\",\"credentials_file\": \"AWS_credentials_file_name\"}, {\"name\": \"sub_account_b\", \"access_key\": \"<access_key>\", \"secret_key\": \"<secret_key>\"}, {\"name\": \"sub_account_c\", \"sts_role\": \"<sts_role>\", \"sts_external_id\": \"<sts_external_id>\"}]}"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 17]
```diff
@@ -0,0 +1,1256 @@
...
+								"value": "application/json"
+							},
+							{
+								"key": "X-chkp-sid",
+								"value": "{{session}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"regions\": [\"eu-west-1\", \"us-east-2\"], \"credentials_file\": \"AWS_credentials_file_name\"}",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:998]: Please change to "raw": "{\"regions\": [\"eu-west-1\", \"us-east-2\"], \"credentials_file\": \"AWS_credentials_file_name\", \"access_key\": \"<access_key>\", \"secret_key\": \"<secret_key>\", \"sts_role\": \"<sts_role>\", \"sta_external_id\": \"<sts_external_id>\", \"scan_gateways\": \"<scan_gateways>\", \"scan_vpn\": \"<scan_vpn>\", \"scan_load_balancers\": \"<scan_load_balancers>\", \"scan_subnets\": \"<scan_subnets>\", \"communities\": [\"<community_a>\", \"<community_b>\"], \"deletion_tolerance\": \"<deletion_tolerance>\", \"sub_accounts\": [{\"name\": \"sub_account_a\",\"credentials_file\": \"AWS_credentials_file_name\"}, {\"name\": \"sub_account_b\", \"access_key\": \"<access_key>\", \"secret_key\": \"<secret_key>\"}, {\"name\": \"sub_account_c\", \"sts_role\": \"<sts_role>\", \"sts_external_id\": \"<sts_external_id>\"}]}",

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 18]
```diff
@@ -0,0 +1,1256 @@
...
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"name\": \"<gwConfigurations_name>\", \"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\", \"vpn_domain\": \"<vpn_domain>\", \"vpn_community\": \"<vpn_community>\", \"deployment_type\": \"<TGW>\", \"tgw_static_routes\": \"<10.0.0.0/16,10.100.0.0/16>\", \"tgw_spoke_routes\": \"<192.168.100.0/24,192.168.200.0/24>\"}"
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:1040]: Please change to "raw": "{\"name\": \"<**gwConfiguration_name**>\", \"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\", \"vpn_domain\": \"<vpn_domain>\", \"vpn_community\": \"<vpn_community>\", \"deployment_type\": \"TGW\", \"tgw_static_routes\": **[\"10.0.0.0/16\", \"10.100.0.0/16\"**], \"tgw_spoke_routes\": **[\"192.168.100.0/24\", \"192.168.200.0/24\"]**}"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 19]
```diff
@@ -0,0 +1,1256 @@
...
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true}, \"related_account\": \"<associated_account>\"}"
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:1076]: Please change to "raw": "{\"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\", \"vpn_domain\": \"<vpn_domain>\", \"vpn_community\": \"<vpn_community>\", \"deployment_type\": \"TGW\", \"tgw_static_routes\": [\"10.0.0.0/16\", \"10.100.0.0/16\"], \"tgw_spoke_routes\": [\"192.168.100.0/24\", \"192.168.200.0/24\"]}"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 20]
```diff
@@ -0,0 +1,1256 @@
...
+						},
+						"url": {
+							"raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/aws/<gwConfigurations_name>",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:1079]: Please change to "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/aws/<**gwConfiguration_name**>"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 21]
```diff
@@ -0,0 +1,1256 @@
...
+							"host": [
+								"{{server}}"
+							],
+							"path": [
+								"v1.8",
+								"cme-api",
+								"v1.2",
+								"gwConfigurations",
+								"aws",
+								"<gwConfigurations_name>"
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:1089]: Please change to "<**gwConfiguration_name**>"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 22]
```diff
@@ -0,0 +1,1256 @@
...
+							},
+							{
+								"key": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"name\": \"<gwConfigurations_name>\", \"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\"}"
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:1198]: Please change to "raw": "{\"name\": \"<**gwConfiguration_name**>\", \"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\"}"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 23]
```diff
@@ -0,0 +1,1256 @@
...
+								"value": "application/json",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\"}"
+						},
+						"url": {
+							"raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/gcp/<gwConfigurations_name>",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:1237]: Please change to "raw": "{{server}}/v1.8/cme-api/v1.2/gwConfigurations/gcp/<**gwConfiguration_name**>",

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 24]
```diff
@@ -0,0 +1,1256 @@
...
+							"host": [
+								"{{server}}"
+							],
+							"path": [
+								"v1.8",
+								"cme-api",
+								"v1.2",
+								"gwConfigurations",
+								"gcp",
+								"<gwConfigurations_name>"
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:1247]: Please change to "<**gwConfiguration_name**>"

File: common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON [hunk 25]
```diff
@@ -0,0 +1,1256 @@
+{
+	"info": {
+		"name": "cme_api_v1.2",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.2.postman_collection.JSON:3]: Please change the name to "cme_api_v1" and override the existing file "common/cme_api_postman/v1/cme_api_v1.postman_collection.json"

File: common/cme_api_postman/v1/cme_api_v1.postman_collection.json [hunk 26]
```diff
@@ -1,1235 +1,1241 @@
...
+                                "key": "Content-Type",
+                                "value": "application/json"
+                            }, {
+                                "key": "X-chkp-sid",
+                                "value": "{{session}}"
+                            }
+                        ],
+                        "body": {
+                            "mode": "raw",
+                            "raw": "{\"regions\" : [\"eu-west-1\", \"us-east-2\"], \"credentials_file\" : \"AWS_credentials_file_name\"}",
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.postman_collection.json:991]: Please update to: "raw": "{\"regions\": [\"eu-west-1\", \"us-east-2\"], \"credentials_file\": \"AWS_credentials_file_name\", \"access_key\": \"<access_key>\", \"secret_key\": \"<secret_key>\", \"sts_role\": \"<sts_role>\", \"sta_external_id\": \"<sts_external_id>\", \"scan_gateways\": \"<scan_gateways>\", \"scan_vpn\": \"<scan_vpn>\", \"scan_load_balancers\": \"<scan_load_balancers>\", \"scan_subnets\": \"<scan_subnets>\", \"communities\": [\"<community_a>\", \"<community_b>\"], \"deletion_tolerance\": \"<deletion_tolerance>\", \"sub_accounts\": [{\"name\": \"sub_account_a\",\"credentials_file\": \"AWS_credentials_file_name\"}, {\"name\": \"sub_account_b\", \"access_key\": \"<access_key>\", \"secret_key\": \"<secret_key>\"}, {\"name\": \"sub_account_c\", \"sts_role\": \"<sts_role>\", \"sts_external_id\": \"<sts_external_id>\"}]}",

File: common/cme_api_postman/v1/cme_api_v1.postman_collection.json [hunk 27]
```diff
@@ -1,1235 +1,1241 @@
...
+                            },
+							{
+                                "key": "Content-Type",
+                                "value": "application/json",
+                                "type": "text"
+                            }
+                        ],
+                        "body": {
+                            "mode": "raw",
+                            "raw": "{\"base64_sic_key\" : \"<encoded_base64_sic_key>\", \"version\" : \"<gw_version>\", \"policy\" : \"<policy_name>\", \"blades\" : {\"ips\" : true, \"anti-bot\" : true, \"url-filtering\" : true}, \"related_account\" : \"<associated_account>\"}"
```
Comment (chkp-natanelm) [inline common/cme_api_postman/v1/cme_api_v1.postman_collection.json:1068]: Please update:
"raw": "{\"base64_sic_key\": \"<encoded_base64_sic_key>\", \"version\": \"<gw_version>\", \"policy\": \"<policy_name>\", \"blades\": {\"ips\":true, \"anti-bot\":true, \"url-filtering\": true, \"https-inspection\": true}, \"related_account\": \"<associated_account>\", \"vpn_domain\": \"<vpn_domain>\", \"vpn_community\": \"<vpn_community>\", \"deployment_type\": \"TGW\", \"tgw_static_routes\": [\"10.0.0.0/16\", \"10.100.0.0/16\"], \"tgw_spoke_routes\": [\"192.168.100.0/24\", \"192.168.200.0/24\"]}"

**Comment by chkp-arielto** [comment]
Thanks @chkp-ccurrier for the contribution! It's appreciated.

**Pull request merged**